internal/talos-test/cmd/root.go

```diff
@@ -0,0 +1,19 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
I have one last nit on this. I think the `talos` portion in the name here is redundant since the code is already part of the Talos repo. But we can change in a follow up.

Reply:
renamed to test-framework